pkg/noderesourcetopology/filter_test.go

```diff
@@ -231,6 +231,66 @@ func TestNodeResourceTopology(t *testing.T) {
 			node:       nodes[0],
 			wantStatus: nil,
 		},
+		{
+			name: "Best effort QoS requesting devices, Pod Scope Topology policy; pod fit",
+			pod: makeBEPodByResourceList(&v1.ResourceList{
+				nicResourceName: *resource.NewQuantity(5, resource.DecimalSI)}),
+			node:       nodes[2],
+			wantStatus: nil,
+		},
+		{
+			name: "Best effort QoS requesting CPU (enough on NUMA) and devices, Pod Scope Topology policy; pod fit",
+			pod: makeBEPodByResourceList(&v1.ResourceList{
```

Comment:
A pod that requests one of memory or cpus is counted as of burstable QoS, will be best-effort only if not specifying any of CPUs and memory. reference: https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/#create-a-pod-that-gets-assigned-a-qos-class-of-burstable 
same note applies for the multiple similar tests below. Maybe also need to adjust the name of makeBEPodByResourceList to not specify BE (as in best effort).

Reply:
Good point, will update.

Reply:
Now that I am rethinking this, we don't actually need a separate function for creating BE or burstable pods. QoS can be influenced by changing the resources passed to `makePodByResourceList`.